### ISSUE TITLE
Screen reader label for project cards

### ISSUE THREAD

**Issue by camdendotlol** [recogito/recogito-client#468]
There should be either some sort of label or a way to suppress this from being read by the screen reader.

<img width="465" height="236" alt="Image" src="https://github.com/user-attachments/assets/524d9937-4839-4194-a2f1-bcd39f677c8a" />

**Comment by camdendotlol** [comment]
Moving to https://github.com/performant-software/cove-recogito/issues/237
